pkg/eventcounter/api/api.go

```diff
@@ -824,6 +826,43 @@ func (s *eventCounterService) GetUserCountV2(
 	}, nil
 }
 
+func (s *eventCounterService) GetMAUCount(
+	ctx context.Context,
+	req *ecproto.GetMAUCountRequest,
+) (*ecproto.GetMAUCountResponse, error) {
+	localizer := locale.NewLocalizer(locale.NewLocale(locale.JaJP))
+	_, err := s.checkRole(ctx, accountproto.Account_VIEWER, req.EnvironmentNamespace, localizer)
+	if err != nil {
+		return nil, err
+	}
+	if req.YearMonth == "" {
+		return nil, localizedError(statusMAUYearMonthRequired, locale.JaJP)
```

Comment:
Can you change `localizedError` to the following implementation?

https://github.com/bucketeer-io/bucketeer/blob/main/pkg/feature/api/feature.go#L64